nixos/modules/tasks/filesystems.nix

```diff
@@ -238,10 +240,10 @@ in
 
         # Filesystems.
         ${concatMapStrings (fs:
-            (if fs.device != null then fs.device
+            (if fs.device != null then escape fs.device
              else if fs.label != null then "/dev/disk/by-label/${fs.label}"
```

Comment:
```fs.label``` to be escaped as well?

Reply:
Can a label have a space in it?

Reply:
seems so
https://stackoverflow.com/questions/15316017/how-do-i-specify-a-label-path-with-spaces-in-etc-fstab

Reply:
@volth thanks for finding that.